src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        String split = ",";
```

Comment:
The string used for splitting values is a good candidate for a constant. It would improve readability and maintainability of your code if it was defined as a private static final constant.